app/models/micropost.rb

```diff
@@ -0,0 +1,18 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  scope :newest, ->{order created_at: :desc}
+  scope :recent_posts, ->{order created_at: :desc}
+  has_one_attached :image
+  ATTR_CHANGE = %i(content image).freeze
```

Comment:
```suggestion
  ATTR_CHANGE = %i(content image).freeze
  
  belongs_to :user
  has_one_attached :image
  
  scope :newest, ->{order created_at: :desc}
```
scope này chỉ khác tên nên chị xóa bớt
em check code lại nhé
và scope thì đặt sau validate nhé cu

Reply:
updated